### PR TITLE
Removed unnecessary call to squeeze

### PR DESCRIPTION
**Pull request by mikerossgithub**
`squeeze` causes a crash when batch size is 1. To reproduce the original error, set batch size for the demo to 1 or 3, or 9 (the later two cause the final batch to have size 1).

### CODE REVIEW

**Comment by matt-gardner** [comment]
Are you using the master branch of allennlp, or version 0.4.2?  We made in a change in master that requires removing the `squeeze` for all batch sizes, so this definitely needs to be updated for the next release.  But I _think_ it's still necessary for 0.4.2.

**Comment by mikerossgithub** [comment]
Yup, I'm using master with pytorch 0.4

**Comment by matt-gardner** [comment]
Ok, thanks for the reminder, but this PR needs to wait until we update this repo for the next release.  If you're running with master, you did the right thing, but this repo is still on 0.4.2.

**Comment by schmmd** [comment]
I'll merge this after #15.

**Review by matt-gardner — APPROVED** [review]

**Comment by schmmd** [comment]
Tests pass and the model trains once master is merged in.

**Pull request merged**
